ci/pipeline.yml

```diff
@@ -1585,10 +1594,6 @@ jobs:
             resource: terraform-yaml-external-production
       - task: check-certificates
         image: general-task
-        file: cg-provision-repo/ci/check-certificates.yml
```

Comment:
This change is unrelated to the rest but the pipeline file was failing local YAML validation for me and this block seemed to be the cause. Seems to have come from #1696. Paging @bengerman13